src/device-registry/models/HealthTips.js

```diff
@@ -69,22 +69,22 @@ tipsSchema.statics = {
 
       switch (args.aqi_category) {
         case "good":
-          modifiedArgs.aqi_category = { min: 0, max: 50 };
+          modifiedArgs.aqi_category = { min: 0, max: 12.09 };
```

Comment:
@Baalmart I was corrected by Richard that we don't calculate AQI index, we use PM2.5 values,
So this just changes these values to match the pm2.5 equivalents